qutip/__init__.py

```diff
@@ -83,6 +62,7 @@
 
 from .core import *
 from .solve import *
+from .solver.brmesolve import *
```

Comment:
Should this not be imported via `from .solve import *` as we do for the rest of the solvers? Or is there a reason to make `brmesolve` special?

Reply:
It is in `qutip.solver` and not in `qutip.solve`. The plan is to do `from .solver import *`, but for now that would import 2 version of `sesolve` and `mesolve`.

Reply:
Okay. Happy with it to stay how it is for now. What is the difference between qutip.solve and qutip.solver? Maybe they need better names.